CSLabs.Api/Proxmox/Responses/NodeStatus.cs

```diff
@@ -5,5 +5,6 @@ public class NodeStatus
         // 0- 100
         public double CpuUsage { get; set; }
         public MemoryUsage MemoryUsage { get; set; }
+        public bool Quorate { get; set; }
```

Comment:
Move this field to a new model called `ClusterStatus` instead since NodeStatus is the status for the node, not the cluster.